src/test/java/no/unit/nva/database/RoleDbTest.java

```diff
@@ -32,50 +35,51 @@ public RoleDbTest() throws InvalidRoleException {
     }
 
     @BeforeEach
-    public void init() {
+    void init() {
         initializeTestDatabase();
         mapper = new DynamoDBMapper(localDynamo);
     }
 
     @Test
-    public void roleDbHasBuilder() {
-        assertDoesNotThrow(RoleDb::newBuilder);
+    void roleDbHasBuilder() {
+        RoleDb.Builder builder = RoleDb.newBuilder();
+        assertNotNull(builder);
```

Comment:
We do not care if it throws Exception in this test. We care that it exists and it returns something not null.

Reply:
Do you need the variable?